config/packages/sulu_admin.yaml

```diff
@@ -1,2 +1,5 @@
 sulu_admin:
     email: "%env(SULU_ADMIN_EMAIL)%"
+    forms:
```

Comment:
I would prepend this configuration in `sulu/sulu`

Reply:
Thought about it as well... But at least to some degree it is project specific, and if we prepend it, it is not possible to remove it anymore... And if one of the configured directories do not exist, then the `Finder` component from Symfony fails. So it would not be possible to use Sulu with another folder structure, therefore I decided against prepending it.

Reply:
@danrot that should be fixable with `->ignoreUnreadableDirs()`

Reply:
Tried it already before, but that only seems to ignore existing directions where the current user is missing some permissions. It still throws an error if the directory does not exist at all.